src/main/java/io/sentry/autoinstall/graphql/GraphqlInstallStrategy.java

```diff
@@ -6,24 +6,27 @@
 import io.sentry.semver.Version;
 import java.util.List;
 import org.eclipse.aether.artifact.Artifact;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class GraphqlInstallStrategy extends AbstractIntegrationInstaller {
-  private static final String GRAPHQL_GROUP = "com.graphql-java";
-  private static final String GRAPHQL_ID = "graphql-java";
-  public static final String SENTRY_GRAPHQL_ID = "sentry-graphql";
+  private static final @NotNull String GRAPHQL_GROUP = "com.graphql-java";
+  private static final @NotNull String GRAPHQL_ID = "graphql-java";
+  public static final @NotNull String SENTRY_GRAPHQL_ID = "sentry-graphql";
 
   public GraphqlInstallStrategy() {
     this(LoggerFactory.getLogger(SentryInstaller.class));
```

Comment:
Not an issue introduced in this PR, but should probably be changed to:

```suggestion
    this(LoggerFactory.getLogger(GraphqlInstallStrategy.class));
```